pkg/testutil/env/path.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2021 The Cockroach Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package env
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+// ExpandPath expands a relative path to a workspace specific one.
```

Comment:
There are two paths that this code follows and this code is critical to testing.  Can you elaborate on why we return one path vs another?  Are you noticed, this code does not work, testing does not work

Reply:
We return the default (bazel test workspace relative) when running with bazel. There are times where it's nice to quickly run tests with go test, like within an editor for example. This allows that to work. I've updated the comments here to reflect that.

To be clear, in CI environments, and most cases really, the bazel path will be used. It seemed like a small concession to make this an option. Are you concerned that something will break here?

Reply:
Nope I like it, just need to document the hell out of the code like you did